src/System.Security.Cryptography.OpenSsl/src/System/Security/Cryptography/RSAOpenSsl.cs

```diff
@@ -155,38 +155,31 @@ public override byte[] Decrypt(byte[] data, RSAEncryptionPadding padding)
             if (padding == null)
                 throw new ArgumentNullException("padding");
 
-            Interop.libcrypto.OpenSslRsaPadding openSslPadding;
-
+            bool useOaepPadding;
             if (padding == RSAEncryptionPadding.Pkcs1)
             {
-                openSslPadding = Interop.libcrypto.OpenSslRsaPadding.RSA_PKCS1_PADDING;
+                useOaepPadding = false;
```

Comment:
A concrete example of why to do things as simple as needed happened just yesterday.

I added a PAL enum for X509Name formatting:
https://github.com/eerhardt/corefx/commit/6c4956feef3e3e672906469dcf87bb68adecca39#diff-443fd6abce16893b65154a678b2e0e23R11

The very same day Jeremy removed it because he took a completely different approach - no longer using X509_NAME_print_ex, but doing it all in managed code:
https://github.com/dotnet/corefx/commit/2f8b7e293c003685a77690e1242fd1a4b5c82c73#diff-443fd6abce16893b65154a678b2e0e23L11

I'll add the PAL enum here now, but I really have no idea how the extra padding options are going to be implemented (i.e. native or managed) until we actually support them.